src/muya/lib/contentState/imageCtrl.js

```diff
@@ -204,6 +204,19 @@ const imageCtrl = ContentState => {
 
     return this.singleRender(outMostBlock, true)
   }
+
+  ContentState.prototype.openImage = function ({ key }, realFilePath) {
+    const block = this.getBlock(key)
+    const { eventCenter } = this.muya
+    if (this.muya.options.openImageWithExternalTool) {
+      this.muya.options.openImageWithExternalTool(realFilePath)
+    }
+
+    this.singleRender(block)
+    eventCenter.dispatch('muya-transformer', { reference: null })
+    eventCenter.dispatch('muya-image-toolbar', { reference: null })
+    return this.muya.dispatchChange()
```

Comment:
I would prefer to move this into the if-condition because it should hide the toolbar if we opened the path?